draft-fossati-tls-attestation.md

```diff
@@ -456,12 +457,15 @@ attestation_channel_binder = {
 ~~~~
 {: #figure-tls-binder title="Format of TLS channel binder."}
 
-* Nonce is the value provided as a challenge by the relying party.
-* The identity key public fingerprint (ik_pub_fingerprint) is a hash of the
+- Nonce is the value provided as a challenge by the relying party.
+- The identity key public fingerprint (ik_pub_fingerprint) is a hash of the
   Subject Public Key Info from the leaf X.509 certificate transmitted in
   the handshake.
-* The channel binder (channel_binder) is a partial transcript of the TLS
-  handshake, up to (but not including) the Certificate message.
+- The channel binder (channel_binder) is a value obtained from the early
+  exporter mechanism offered by the TLS implementation (see section 7.5 of
+  {{RFC8446}}). This Early Exporter Value (EEV) must be obtained immediately
+  following the ServerHello message, using 'attestation-binder' as the label
+  and an empty context.
```

Comment:
I think we should be more formal here, using the notation of RFC 8446, Sec. 7.5.

Reply:
Alright, I've tweaked this bulletpoint a bit (thanks for the suggestion, I realised I hadn't nailed down the binder length) and added a figure to show how the call should look like. Let me know if that seems in line with what you expected!

Reply:
@ionut-arm Yep, this is much better. I just think that the context is an empty string (""), not an empty list ([]).